### DISCUSSION TITLE
Query results on the mesh

### DISCUSSION THREAD

**Discussion by abarun22** [idaholab/moose#16250] [Q&A General] [answered]
Dear all,
I was wondering if there is an option in MOOSE to probe the nodal values in the mesh during the post-processing stage. I suppose this could be a part of feature in peacock which i am not aware of. I would like to perform a node based comparison of resultants with ANSYS. Please could any body let me know how this can be done?
Kind regards,
Arun

**Comment by jessecarterMOOSE** [comment, marked as answer]
There is a vector postprocessor called NodalValueSampler that might help.
[https://mooseframework.inl.gov/source/vectorpostprocessors/NodalValueSampler.html](https://mooseframework.inl.gov/source/vectorpostprocessors/NodalValueSampler.html)
